flytekit/core/context_manager.py

```diff
@@ -365,7 +365,7 @@ def get(
         fpath = self.get_secrets_file(group, key, group_version)
         v = os.environ.get(env_var)
         if v is not None:
-            return v
+            return v.strip()
```

Comment:
why removes `strip()`?

Reply:
I think it's because of this PR.
https://github.com/flyteorg/flytekit/pull/2077/files

Reply:
dropped 😅